cljfmt/src/cljfmt/core.cljc

```diff
@@ -390,3 +391,19 @@
          (reformat-form (cond-> options
                           alias-map (assoc :alias-map alias-map)))
          (n/string)))))
+
+(def default-linebreak
```

Comment:
Maybe rename this to `default-line-separator`, and move it to `cljfmt.diff` where it's being used.

Reply:
renamed. it's also being used indirectly by main via (now renamed) `find-line-separator`